diff -ru glibc-2.27.orig/debian/sysdeps/linux.mk glibc-2.27/debian/sysdeps/linux.mk
--- glibc-2.27.orig/debian/sysdeps/linux.mk	2018-11-14 14:11:51.000000000 +0000
+++ glibc-2.27/debian/sysdeps/linux.mk	2018-11-14 14:14:49.000000000 +0000
@@ -1,7 +1,7 @@
 # When changing this, make sure:
 # - to update debian/debhelper.in/libc.preinst!
 # - that all builds for a given gnu triplet have the same minimum version (see bug#882255)
-MIN_KERNEL_SUPPORTED := 3.2
+MIN_KERNEL_SUPPORTED := 3.0
 libc = libc6
 
 # Do not build pt_chown on this platform
Only in glibc-2.27/debian/sysdeps: linux.mk~
diff -ru glibc-2.27.orig/sysdeps/unix/sysv/linux/configure glibc-2.27/sysdeps/unix/sysv/linux/configure
--- glibc-2.27.orig/sysdeps/unix/sysv/linux/configure	2018-11-14 14:11:52.000000000 +0000
+++ glibc-2.27/sysdeps/unix/sysv/linux/configure	2018-11-14 14:19:34.000000000 +0000
@@ -8,13 +8,13 @@
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking installed Linux kernel header files" >&5
 $as_echo_n "checking installed Linux kernel header files... " >&6; }
-if ${libc_cv_linux320+:} false; then :
+if ${libc_cv_linux300+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <linux/version.h>
-#if !defined LINUX_VERSION_CODE || LINUX_VERSION_CODE <  (3 *65536+ 2 *256+ 0) /* 3.2.0 */
+#if !defined LINUX_VERSION_CODE || LINUX_VERSION_CODE <  (3 *65536+ 0 *256+ 0) /* 3.0.0 */
 # error kernel headers missing or too old
 #endif
 int
@@ -26,20 +26,20 @@
 }
 _ACEOF
 if ac_fn_c_try_compile "$LINENO"; then :
-  libc_cv_linux320='3.2.0 or later'
+  libc_cv_linux300='3.0.0 or later'
 else
-  libc_cv_linux320='missing or too old!'
+  libc_cv_linux300='missing or too old!'
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $libc_cv_linux320" >&5
-$as_echo "$libc_cv_linux320" >&6; }
-if test "$libc_cv_linux320" != '3.2.0 or later'; then
+{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $libc_cv_linux300" >&5
+$as_echo "$libc_cv_linux300" >&6; }
+if test "$libc_cv_linux300" != '3.0.0 or later'; then
   as_fn_error $? "GNU libc requires kernel header files from
-Linux 3.2.0 or later to be installed before configuring.
+Linux 3.0.0 or later to be installed before configuring.
 The kernel header files are found usually in /usr/include/asm and
 /usr/include/linux; make sure these directories use files from
-Linux 3.2.0 or later.  This check uses <linux/version.h>, so
+Linux 3.0.0 or later.  This check uses <linux/version.h>, so
 make sure that file was built correctly when installing the kernel header
 files.  To use kernel headers not from /usr/include/linux, use the
 configure option --with-headers." "$LINENO" 5
@@ -51,7 +51,7 @@
 # set arch_minimum_kernel already, let that override our defaults here.
 # Note that we presume such a fragment has set libc_cv_gcc_unwind_find_fde
 # if appropriate too.
-test -n "$arch_minimum_kernel" || arch_minimum_kernel=3.2.0
+test -n "$arch_minimum_kernel" || arch_minimum_kernel=3.0.0
 if test -n "$minimum_kernel"; then
 
   user_version=$((`echo "$minimum_kernel.0.0.0" | sed 's/\([0-9]*\)\.\([0-9]*\)\.\([0-9]*\).*/\1 \* 65536 + \2 \* 256 + \3/'`))
diff -ru glibc-2.27.orig/sysdeps/unix/sysv/linux/configure.ac glibc-2.27/sysdeps/unix/sysv/linux/configure.ac
--- glibc-2.27.orig/sysdeps/unix/sysv/linux/configure.ac	2018-11-14 14:11:52.000000000 +0000
+++ glibc-2.27/sysdeps/unix/sysv/linux/configure.ac	2018-11-14 14:20:37.000000000 +0000
@@ -1,7 +1,7 @@
 GLIBC_PROVIDES dnl See aclocal.m4 in the top level source directory.
 # Local configure fragment for sysdeps/unix/sysv/linux.
 
-define([LIBC_LINUX_VERSION],[3.2.0])dnl
+define([LIBC_LINUX_VERSION],[3.0.0])dnl
 if test -n "$sysheaders"; then
   OLD_CPPFLAGS=$CPPFLAGS
   CPPFLAGS="$CPPFLAGS $SYSINCLUDES"
@@ -33,7 +33,7 @@
 # set arch_minimum_kernel already, let that override our defaults here.
 # Note that we presume such a fragment has set libc_cv_gcc_unwind_find_fde
 # if appropriate too.
-test -n "$arch_minimum_kernel" || arch_minimum_kernel=3.2.0
+test -n "$arch_minimum_kernel" || arch_minimum_kernel=3.0.0
 if test -n "$minimum_kernel"; then
   changequote(,)
   user_version=$((`echo "$minimum_kernel.0.0.0" | sed 's/\([0-9]*\)\.\([0-9]*\)\.\([0-9]*\).*/\1 \* 65536 + \2 \* 256 + \3/'`))
